content/data-types/reading/some-java-practice/_index.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Some Java Practice"
+date: 2023-06-14T09:27:57-05:00
+draft: false
+weight: 1
+originalAuthor: Courtney Frey # to be set by page creator
+originalAuthorGitHub: Courtney Frey # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Let’s move beyond our “Hello, World” example and explore a simple temperature conversion program. We want our function to convert a Fahrenheit temperature to Celsius.
+
+## Temperature Conversion
+
+1. Open the `TempConverter` file in the `java-web-dev-exercises` project in IntelliJ.
```

Comment:
Section needs reworking and link to project https://github.com/LaunchCodeEducation/java-web-dev-projects-17.  Also update image.
<img width="502" alt="tempConversion" src="https://github.com/LaunchCodeEducation/java-web-dev-curriculum/assets/39868164/c1d37686-c858-4bcf-8560-dca2b509a512">

Reply:
new repo - new screen shots